Bottom: 0b3373d787027a124aa23ea01f8b1d011e3023da
Top:    b0268e10139f2503f831c8ecd9ffbbafca4da522
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-08-31 11:27:04 +0800

Refresh of vhost_net-basic-batch

---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index daeec47..4ca89b3 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -2500,7 +2500,6 @@ int vhost_prefetch_desc_indices(struct vhost_virtqueue *vq,
 		last_avail_idx = (last_avail_idx + 1) & (vq->num - 1);
 	}
 
-	total = ret;
 	last_used_idx = vq->last_used_idx & (vq->num - 1);
 	while (total) {
 		copied = min((u16)(vq->num - last_used_idx), total);
@@ -2518,7 +2517,6 @@ int vhost_prefetch_desc_indices(struct vhost_virtqueue *vq,
 		total -= copied;
 	}
 
-out:
 	/* Only get avail ring entries after they have been exposed by guest. */
 	smp_rmb();
 	return ret;
